CHANGELOG.md

```diff
@@ -0,0 +1,58 @@
+## Change Log
+
+### Unreleased
+
+This release has a number of bug fixes in addition to a few new features.
+Following a complete transition to Python 3, with dropped Python 2 support,
+major work was made towards code modernization and quality.
+
+- The code is now Black formatted and Flake8 tested
+- Greatly improved unittest framework
+- Embedded PLY version updated to 3.11
+- New option: `--no-embed-preamble` create separate files for preamble and
+  loader instead of embedding in each output file
+- New option: `--allow-gnu-c` do not undefine `__GNUC__`
+- Fix library loader search path on macOS
+- Fix rare bug, processing (legacy) header files with MacRoman encoding on macOS
+- Add full support for floating and integer constants
+- Add support for sized integer types on Windows
+- Add support to handle restrict keyword
+- Add name formats to posix library loader
```

Comment:
Maybe past tense would be better (`fixed`, `added`, etc.)?

Reply:
Good, updated.